web/client/utils/CoordinatesUtils.js

```diff
@@ -56,6 +57,100 @@ let crsLabels = {
     "EPSG:3857": "EPSG:3857"
 };
 
+const normalizePoint = (point) => {
+    return {
+        x: point.x || 0.0,
+        y: point.y || 0.0,
+        srs: point.srs || 'EPSG:4326'
+    };
+};
+
+const reproject = (point, source, dest, normalize = true) => {
```

Comment:
You moved this, did you change anything?

Reply:
no, i didn't change the function